addon/commands/move-article-command.js

```diff
@@ -71,7 +71,8 @@ export default class MoveArticleCommand {
       articleContainerElement.getMaxOffset()
     );
     const articleHtml = articles.reduce(
-      (html, article) => (html += article.boundNode.outerHTML),
+      (html, article, index) =>
+        (html += this.generateArticleHtml(article, index)),
```

Comment:
couldn't we leverage the insert-article command here and limit the logic needed to generate the html?

Reply:
The problem is that the insert article command as it name says inserts the article, and I would like to keep everything in one insertion for performance reasons

Reply:
I could export the generateArticleHtml function to a common file, but I think they are different enough now for it to not be worth it

Reply:
fair enough, might make sense to include that in a comment

Reply:
The part of one insertion per operation or the exporting the function one?